pkg/util/util.go

```diff
@@ -198,6 +198,21 @@ func CompileRegexps(regexpStrings []string) ([]*regexp.Regexp, error) {
 	return regexps, nil
 }
 
+// Detects if using systemd as the init system
+// Please note that simply reading /proc/1/cmdline can be misleading because
+// some installation of various init programs can automatically make /sbin/init
+// a symlink or even a renamed version of their main program.
+// TODO(dchen1107): realiably detects the init system using on the system:
+// systemd,
+func UsingSystemdInitSystem() bool {
+	_, err := os.Stat("/run/systemd/system")
```

Comment:
This should be written as:

if _, err := os.Stat(...); err != nil {
  return false
}
return true